### PR TITLE
Hw 7

### PR DESCRIPTION
**Pull request by abdulaziztag**
I didnt add footer, because in figma there is no footer

### CODE REVIEW

File: client/src/app/pages/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+@Injectable()
+export class LoginComponent {
+  constructor(private http: HttpClient) {}
+  login() {
+    let data = this.http
```
Comment (devbarden) [inline client/src/app/pages/login/login.component.ts:13]: why 'let' ?

File: client/src/app/pages/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      .get('https://jsonplaceholder.typicode.com/todos/1')
+      .subscribe()
+    console.log(data)
```
Comment (devbarden) [inline client/src/app/pages/login/login.component.ts:16]: please remove logs from FE

**Review by devbarden — APPROVED** [review]
just a couple of issues

File: client/src/app/pages/login/login.component.ts (hunk 2 above)
Comment (abdulaziztag) [reply client/src/app/pages/login/login.component.ts:16]: Ok, I just checked the script and forgot to delete logs((

File: client/src/app/pages/login/login.component.ts (hunk 1 above)
Comment (abdulaziztag) [reply client/src/app/pages/login/login.component.ts:13]: Old habit))

**Pull request merged**